apps/frontend/src/hooks/use-active-wallet.ts

```
@@ -12,7 +12,18 @@ type ActiveWalletResult = {
 
 export function useActiveWallet(): ActiveWalletResult {
   const { address: freighterAddress, signXDR } = useWallet();
-  const { isAuthenticated, wallet: pollarWallet, getClient } = usePollar();
+  const pollarContext = (() => {
+    try {
+      return usePollar();
```

ESLint

[error] React Hook "usePollar" cannot be called inside a callback. React Hooks must be called in a React function component or a custom React Hook function.

(react-hooks/rules-of-hooks)